src/rocks_global_options.cpp

```diff
@@ -81,6 +81,11 @@ namespace mongo {
                                moe::Bool,
                                "If true, we will turn on RocksDB's advanced counters")
             .setDefault(moe::Value(false));
+        rocksOptions
+            .addOptionChaining("storage.rocksdb.singleDeleteIndex",
+                               "rocksdbSingleDeleteIndex", moe::Bool,
+                               "If true, RocksDB will use SingleDelete method for unindex")
```

Comment:
For now let's just add a warning "This is still experimental. Use this only if you know what you're doing"